policyengine_us/parameters/gov/usda/snap/income/deductions/utility/limited/by_household_size/amount.yaml

```
@@ -0,0 +1,34 @@
+description: The US department of agriculture provides the following limited utility allowance, baed on state and household size.
```

Since these are state-defined parameters (in the future we will move these parameters to state folders).
```suggestion
description: States that define Supplemental Nutrition Assistance Program limited utility allowances by household size set these allowance values.
```